command/snapshot_command.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+// SnapshotCommand is a Command implementation that just shows help for
+// the subcommands nested below it.
+type SnapshotCommand struct {
+	Ui cli.Ui
+}
+
+func (c *SnapshotCommand) Run(args []string) int {
+	return cli.RunResultHelp
+}
+
+func (c *SnapshotCommand) Help() string {
+	helpText := `
+Usage: consul snapshot <subcommand> [options] [args]
```

Comment:
It would be nice to have a `consul snapshot inspect` to view metadata about a snapshot, like the indexes, checksums, etc. This way if you have multiple snapshots you can help disambiguate.